lightkurve/search.py

```diff
@@ -20,7 +20,7 @@
 
 log = logging.getLogger(__name__)
 
-__all__ = ['open', 'search_targetpixelfile', 'search_lightcurvefile']
+__all__ = ['search_targetpixelfile', 'search_lightcurvefile', 'search_cutout', 'open']
```

Comment:
I'm gonna suggest again that this should be called `search_TESScutout`. My concern is that the majority of TESS users won't notice that this feature exists, and anyone who tab completes and sees `search_cutout` will have no idea what it really does. 

I think we need a more informative name.

Reply:
In a GO office meeting, we decided on the name `search_tesscut()`.